src/routes/projects/[slug].svelte

```diff
@@ -1,7 +1,10 @@
 <script context="module">
+  import { page } from '$app/stores'
+  import Head from '$lib/components/head.svelte'
   import { client } from '$lib/graphql-client'
   import { projectQuery } from '$lib/graphql-queries'
   import { marked } from 'marked'
+  import { siteMetadataStore } from '../../stores/site-metadata'
```

Comment:
Can this not be `$stores/site-metadata`?

Reply:
Jep! I'll get that done, thanks